### PR TITLE
Slight 2-2-dev cleanup

### PR DESCRIPTION
**Pull request by athal7**

### CODE REVIEW

**Comment by jordan-brough** [comment]
:+1: assuming specs pass

**Comment by athal7** [comment]
seem to be having some issue with the specs on this repo. circleci wants to run `rake db:create` when it usually doesn't want to for spree extensions

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+database:
+  override:
+    - bundle exec rake test_app
```
Comment (jordan-brough) [inline circle.yml]: it looks like circleci is now running `bundle exec rake test_app` twice -- https://circleci.com/gh/bonobos/spree_avatax/16 ?  Do you know of a way to instead just make this a no-op or something?

Comment (athal7) [inline circle.yml]: removed the second one now (it was configured in the UI), thanks for the heads up

Comment (jordan-brough) [inline circle.yml]: Looks great now. Thanks!

**Pull request merged**